modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationsInTheAPIBuilder/EditAPIApplication.testcase

```diff
@@ -29,6 +29,245 @@ definition {
 		}
 	}
 
+	@priority = 4
+	test CanEditSchemaNameAndDescriptionInPublishedAPIApplication {
+		property portal.acceptance = "true";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("Given create an published API application with related schema") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Schemas",
+				title = "App-test");
+
+			Click(locator1 = "Button#PLUS");
+
+			Type(
+				locator1 = "TextInput#NAME",
+				value1 = "Name default");
+
+			Type(
+				locator1 = "TextInput#DESCRIPTION_FIELD",
+				value1 = "Description default");
+
+			Click(locator1 = "ObjectAdmin#OBJECT_DROPDOWN");
+
+			Click(
+				key_value = "APIApplication",
+				locator1 = "ObjectAdmin#ADD_ENTRY_DROPDOWN");
+
+			Click(locator1 = "APIBuilder#CREATE_BUTTON");
+
+			Alert.viewSuccessMessage();
+
+			Click.pauseClickAt(locator1 = "APIBuilder#PUBLISH_BUTTON");
+		}
+
+		task ("And Given edit the schema in Edit API Application > Schemas") {
+			APIBuilderUI.goToEditAPIApplicationPage(key_title = "App-test");
+
+			Click(
+				locator1 = "Button#BUTTON_WITH_VALUE",
+				value = "Schemas");
+
+			Click(
+				key_title = "Name default",
+				locator1 = "APIBuilder#DROPDOWN_MENU");
+
+			Click(locator1 = "APIBuilder#EDIT_BUTTON");
+
+			Type(
+				locator1 = "TextInput#NAME",
+				value1 = "Name edited");
+
+			Type(
+				locator1 = "TextInput#DESCRIPTION_FIELD",
+				value1 = "Description edited");
+		}
+
+		task ("When I publish changed name and description") {
+			Click.pauseClickAt(locator1 = "APIBuilder#PUBLISH_BUTTON");
+		}
+
+		task ("Then I can see success alert message about API schema changes were published And Then I can see updated name and description in Info tab") {
+			Click(locator1 = "CommerceNavigation#BACK");
```

Comment:
We can use the following to assert the success alert message
`Alert.viewSuccessMessage();`